openshift/templates/images/nodejs/Dockerfile

```diff
@@ -1,12 +1,12 @@
 # Base alpine images with node.js
-FROM node:18.14-alpine3.17
+FROM node:lts-alpine3.17
```

Comment:
I think we need to specify the version here. My understanding is that the lts-alpine3.17 image will keep up with whatever the latest lts version is so it won't point to 18.18.0 at some point in the future.